src/Synapse/Synapse/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Added example for definition file structure of 'New-AzSynapseManagedPrivateEndpoint'
```

Comment:
No need for change log if only docs changes.

Reply:
ok, thanks, reverted.